### DISCUSSION TITLE
Login issue with MLSA ID.

### DISCUSSION THREAD

**Discussion by theyashsisodiya** [microsoft/SATechnicalOnboarding#93] [Questions]
Removing the post for privacy concerns.

**Comment by PranjaliBhardwaj** [comment]
hey! I have question when i login to my azure student account, it says Unable to confirm your University ID, I have seen this on github too, but that isnt helping. I want to ask you, if you can help, I ahve no subscription of azure neither it letting me login, but when i hit sign up, it open up with my student ambassador account. Is there anything i should worry about? What should I do?
![Screenshot 2024-01-22 021132](https://github.com/microsoft/SATechnicalOnboarding/assets/146981751/64b93d2e-d8a0-49e3-a605-b58625277162)

**Reply by PranjaliBhardwaj** [reply]
Like im new to mlsa, joined this week and have no clue, im right now off to my technical onboarding . Help

**Reply by AnthonyByansi** [reply]
There is a technical issue with azure for students which is being resolved. I would recommend you attain your Alpha milestone and redeem the benefits you're entailed to [Here](https://my.visualstudio.com/)

![image](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/cb4f5476-db84-4201-a25e-98f2c1bf7b85)

**Reply by AnthonyByansi** [reply]
And you can always start off with [These instructions](https://github.com/microsoft/SATechnicalOnboarding/blob/main/technical-onboarding-instructions.md) for your technical onbaording

**Comment by theyashsisodiya** [comment]
It's because you are on NEW MLSA you have to become Alpha for free Azure Activation. how to become alpha ? login your MLSA ID on Microsoft learning complete 2-3 Learning path again learning path not modules. wait till next day you will be promoted to MLSA Alpha,
________________________________

**Reply by theyashsisodiya** [reply]
@PranjaliBhardwaj

**Comment by PranjaliBhardwaj** [comment]
Without Technical onboarding?

**Reply by AnthonyByansi** [reply]
Technical onboarding is the process where a new student ambassador is helped to understand what excellence means at Microsoft and clear goals are defined to help you achieve it. 
You are required to complete a few tasks to familiarize yourself with some of the technologies like how to use github codespaces to ease your development among others.

**Comment by theyashsisodiya** [comment]
Yes

On Jan 22, 2024 11:23 AM, Pranjali Bhardwaj ***@***.***> wrote:

Without Technical onboarding?

**Reply by AnthonyByansi** [reply]
Hi @theyashsisodiya could please provide clear detail of the issue you are facing so we could help? Could you take a screenshot of the issue you want help with?

**Comment by theyashsisodiya** [comment]
Here is the video demonstration of the issue which I am facing : https://stdntpartners-my.sharepoint.com/:v:/g/personal/yashraj_singhsisodiya_studentambassadors_com/EZebb3GTXDFOnyl2COdQtPIBHme0cCfFBoXUPjGQSjsa6A?e=daAMAh

But I don't know. Somehow I am able to track my progress like I can see my alpha Badge there but This is the id which I used to apply for the mlsa.

And one more issue is I didnt get mail of Linkedin learning.
________________________________

**Reply by AnthonyByansi** [reply]
An email will be sent to you with all the links to access your benefits, Linkedln learning inclusinve in 2-3 weeks

**Comment by theyashsisodiya** [comment]
Yeah, I have completed the on boarding process Now the only things remaining that I need to post a challenge.

My issue is I didn't get the free linked Learning and And Other one is login redirect mail. Watch this video for better understanding what I'm trying to say (https://stdntpartners-my.sharepoint.com/:v:/g/personal/yashraj_singhsisodiya_studentambassadors_com/EZebb3GTXDFOnyl2COdQtPIBHme0cCfFBoXUPjGQSjsa6A?e=daAMAh)

________________________________

**Reply by AnthonyByansi** [reply]
You being logged in with the email you used to apply for MLSA is a normal process now, @SalemTracey will have this addressed

**Comment by theyashsisodiya** [comment]
Sure, thank you sir. Please also look at the login issue.

________________________________

**Comment by theyashsisodiya** [comment]
Ohh, thank you I have seen some youtube video but they have logged in with their mls ID. But I am logged in with that id which I used to apply for mlsa. This is like confusing for me. That's why

Thank you again
________________________________

**Comment by SalemTracey** [comment]
@theyashsisodiya - i removed the content of your original post out of privacy concerns. I understand you are working with a member of the operations team to resolve.
